packages/action/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "action",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```

Comment:
Let's add a linter to this library, like we have for the API. I suggest we use the same one:

https://github.com/GoogleCloudPlatform/flaky-service/blob/master/packages/api/package.json#L9

This can then just be our `test`.